x/gov/client/cli/prompt.go

```diff
@@ -76,22 +76,19 @@ func Prompt[T any](data T, namePrefix string) (T, error) {
 
 		// create prompts
 		prompt := promptui.Prompt{
-			Label:    fmt.Sprintf("Enter %s's %s", namePrefix, strings.ToLower(client.CamelCaseToString(v.Type().Field(i).Name))),
+			Label:    fmt.Sprintf("Enter %s %s", namePrefix, strings.ToLower(client.CamelCaseToString(v.Type().Field(i).Name))),
 			Validate: client.ValidatePromptNotEmpty,
 		}
 
 		fieldName := strings.ToLower(v.Type().Field(i).Name)
-		// validation per field name
```

Comment:
Removed this as this can collide with `msg type url` which is definitely not a web url.